src/ssm.c

```diff
@@ -113,21 +113,36 @@ static void Check_Error(BMS_INPUT_T *input, BMS_STATE_T *state, BMS_OUTPUT_T *ou
     // }
     //[TODO] change to case statement
     if (state->curr_mode != BMS_SSM_MODE_INIT) {
-        int16_t max_cell_temp_thres_C = state->pack_config->max_cell_temp_dC;
```

Comment:
why are you returning on every one of these errors. Each tick you are only allowing one error to be asserted?

Whey are there no calls to Error_Pass for any of these?